src/CustomPostType/Capability.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Bright Nucleus Custom Content.
+ *
+ * Config-driven WordPress Custom Post Types And Custom Taxonomies.
+ *
+ * @package   BrightNucleus\CustomContent
+ * @author    Alain Schlesser <alain.schlesser@gmail.com>
+ * @license   MIT
+ * @link      https://www.brightnucleus.com/
+ * @copyright 2016 Alain Schlesser, Bright Nucleus
+ */
+
+namespace BrightNucleus\CustomContent\CustomPostType;
+
+/**
+ * Class Capability.
+ *
+ * Constants to use for setting the custom post type's capabilities.
+ *
+ * @since   0.1.0
```

Comment:
This will go into v0.1.8.
```suggestion
 * @since   0.1.8
```

Reply:
Thanks for fixing that. C&P mistake